src/rf/rf/urls.py

```diff
@@ -19,8 +20,11 @@
 urlpatterns = patterns(
     '',
     url(r'^', include(apps.home.urls)),
+    url(r'^', include(apps.core.urls)),
     url(r'^api/', include(router.urls)),
```

Comment:
Actually, I'm thinking about adding `api/` as a prefix to all REST endpoint URLs to distinguish them from web app URLs which may have the same structure.